### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DajmondFM**
Added version for aw-server-rust

### CODE REVIEW

**Comment by 2e3s** [comment]
Hello! Thank you for your attentiveness, I think this can be changed easier with less copypaste by changing

> Add the following section to your `aw-server.toml` file in [config directory](https://docs.activitywatch.net/en/latest/directories.html#config):

to 
> Add the following section to your `aw-server/aw-server.toml` or `aw-server-rust/config.toml` file in [config directory](https://docs.activitywatch.net/en/latest/directories.html#config):

**Comment by DajmondFM** [comment]
Hi! This would be easier but you also need to change the windows path (idk about others) from \ to \\

**Comment by 2e3s** [comment]
I think this is true for both cases, the current Windows path is incorrect in the example and needs to be changed to `\\`. Or better yet to use single quotes:
```toml
aw-watcher-media-player = 'C:\Users\<USER>\AppData\Local\aw-watcher-media-player\visualization'
```

**Comment by DajmondFM** [comment]
Yeah i think using single quotes will work the best

File: README.md [hunk 1]
```diff
@@ -91,13 +91,13 @@ Use `-vv` to see what's reported.
 This watcher has a visualization which attempts to do its best to display the sorted list of artists with the overall play time for each artist.
 Note that ActiveWatch UI gives no abilities for the widget to control its sizing, so it may appear smaller than builtin visualizations.
 
-1. Add the following section to your `aw-server.toml` file in [config directory](https://docs.activitywatch.net/en/latest/directories.html#config):
+1. Add the following section to your aw-server/aw-server.toml or aw-server-rust/config.toml file in [config directory](https://docs.activitywatch.net/en/latest/directories.html#config):
```
Comment (2e3s) [inline README.md]:
```suggestion
1. Add the following section to your `aw-server/aw-server.toml` or `aw-server-rust/config.toml` file in [config directory](https://docs.activitywatch.net/en/latest/directories.html#config):
```

**Comment by 2e3s** [comment]
Thanks!

**Pull request merged**